operator/redisfailover/service/generator.go

```diff
@@ -563,6 +506,19 @@ func generateSentinelNetworkPolicy(rf *redisfailoverv1.RedisFailover, labels map
 					Ports: ports,
 				},
 			},
+			Egress: []np.NetworkPolicyEgressRule{
+				np.NetworkPolicyEgressRule{
+					To: []np.NetworkPolicyPeer{
+						np.NetworkPolicyPeer{
+							NamespaceSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"app.kubernetes.io/instance": namespace,
+								},
+							},
+						},
+					},
```

Comment:
What happens if multiple RedisFailover CRs exist in the same namespace? Should we further restrict communication so that a sentinel can ONLY communicate with resources owned by its RedisFailover? _Maybe_ something like this?

```diff
To: []np.NetworkPolicyPeer{
	np.NetworkPolicyPeer{
		NamespaceSelector: &metav1.LabelSelector{
			MatchLabels: map[string]string{
				"app.kubernetes.io/instance": namespace,
			},
		},
+		PodSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"redisfailovers.databases.spotahome.com/name": rf.Name,
+			},
+		},
	},
},
```

Reply:
sentinels NetworkPolicy will only retain `PodSelector` for egress rule, as `PodSelector` assumes it is within a namespace location.